custom-connectors/AzureADPhoneMethods/apiProperties.json

```diff
@@ -41,6 +41,7 @@
     },
     "iconBrandColor": "#adc7db",
     "capabilities": [],
-    "publisher": "Jan Bakker"
+    "publisher": "Jan Bakker",
+    "stackOwner": "Jan Bakker"
```

Comment:
@BakkerJan Publisher and StackOwner values should not be individual names, these should be a company or service names